postprocessing/processors/ReductionProcessor.py

```diff
@@ -0,0 +1,119 @@
+from .base_processor import BaseProcessor
```

Comment:
module names should be all lowercase, please rename the file `reduction_processor.py`